eval/kiali-ossm/scenarios/fix_bookinfo_fault_injection/setup.sh

```
@@ -0,0 +1,49 @@
+# Removes leftover resources — including any AuthorizationPolicies created by a
+# previous agent run — so the agent starts with a clean slate each time.
+echo "Removing existing ratings Istio resources…"
+$KUBECTL delete virtualservice     ratings                  -n "$NAMESPACE" --ignore-not-found
+$KUBECTL delete destinationrule    ratings                  -n "$NAMESPACE" --ignore-not-found
+$KUBECTL delete peerauthentication ratings-permissive-mtls  -n "$NAMESPACE" --ignore-not-found
+# Remove AuthorizationPolicies targeting ratings that a previous agent may have created
+$KUBECTL delete authorizationpolicy allow-reviews-to-ratings  -n "$NAMESPACE" --ignore-not-found || true
+$KUBECTL delete authorizationpolicy ratings-viewer             -n "$NAMESPACE" --ignore-not-found || true
+$KUBECTL get authorizationpolicy -n "$NAMESPACE" --no-headers 2>/dev/null \
+  | grep -i ratings | awk '{print $1}' \
+  | xargs -r $KUBECTL delete authorizationpolicy -n "$NAMESPACE" --ignore-not-found || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)